src/coreclr/tools/r2rdump/CoreDisTools.cs

```diff
@@ -29,9 +29,6 @@ public enum TargetArch
         [DllImport(_dll, CallingConvention = CallingConvention.Cdecl)]
         public static extern IntPtr InitBufferedDisasm(TargetArch Target);
 
-        [DllImport(_dll, CallingConvention = CallingConvention.Cdecl)]
```

Comment:
I've a question here. Why are we removing this? Is it no longer used anywhere?

Reply:
[Yes](https://github.com/search?q=repo%3Adotnet%2Fruntime+DumpCodeBlock+language%3AC%23&type=code&l=C%23)

Reply:
Ok I see. I kicked off an outerloop run just to be safe this entire PR is good to go. If it returns all green, then we can approve and merge this.